### ISSUE TITLE
Page Not Found

### ISSUE THREAD

**Issue by Joehurley** [nicolaskruchten/jupyter_pivottablejs#4]
Love this idea...but can't quite get it to work. I am relatively new to python, so I am assuming I am missing a setting, version conflict, or maybe a firewall issue (or something similar). I keep getting an error:
404 : Not Found
You are requesting a page that does not exist!

in my ipython notebook. I don't see anything on github to explain this, but have a feeling it is something simple. Any help would be greatly appreciated! Thanks.
![pivottablejs_error](https://cloud.githubusercontent.com/assets/7088183/9878767/03ca4f0c-5b8a-11e5-822b-5794366d7ce3.jpg)

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! Can you tell me which versions of Python, Jupyter and Pandas you're using? I should be able to get a fix together :)

**Comment by Joehurley** [comment]
python version is 2.7.7
pandas version is 0.16.2
jupyter/ipython nb = 3.2.0

**Comment by nicolaskruchten** [comment]
Hmm, I'm having trouble replicating this issue, even with the same versions of everything. Are you still seeing this? Were you ever able to get it to work?

**Comment by Joehurley** [comment]
I appreciate you trying to figure it out. Much to my chagrin, I have not figured it out either, I get the same error as above...strange. I upgraded through pip as well. I also tried with a number of generic dfs  (versus reading one from a database I use):

df = DataFrame({'one' : Series([1.4, 2.2, 3.3, 5.1]), 'two' : Series([1.4, 2.3, 3.1, 4.9])})

Do indexes matter at all? This is all on my work machine...maybe there is some issue? I will try at home and see if I can get it to work. Any other ideas would be appreciated, but not required. Great idea...would LOVE to get this working. Thanks!

**Comment by nicolaskruchten** [comment]
OK, we can crack this! :)

A few questions:

1. what OS/Browser are you using?
2. is the notebook you are working from in the top-level of your notebook directory or in a subdirectory?
3. what are the permissions in the directory of your notebook?

Thanks!

**Comment by Joehurley** [comment]
Ha! I really appreciate your willingness to get me going. I am actually
getting a new machine next week. Before I waste your time, let me reinstall
everything on my new machine, and see if i can get it to work. If not, I
will let you know! Thanks again...

**Comment by nicolaskruchten** [comment]
OK! Although if I could still get the answers to these questions I might be able to solve it for others as well, so if you have a chance, in case you're not the only one to see this bug :)

**Comment by Joehurley** [comment]
OK. New machine...same issue! (i) I am using windows and chrome. (ii) it is top level of notebook directory. (iii) i am not sure about the permission question. my directory is simply on my hard drive.

**Comment by Joehurley** [comment]
OK. Figured it out...with a clue from your previous response...so thanks. In my ipython_config file, I set the notebook directory to be within my self-defined project directory (i.e. c.NotebookApp.notebook_dir = 'project_direct/notebooks'). It appears that installing pivottablejs puts the pivottabljs.html file in the project directory that I chdir into within the config file...not necessarily the ipython NOTEBOOK directory. So...my bad. Moving pivottblejs.html to notebook directory, it works! Hopefully, you didn't spend too much time, but it works now, and it is awesome. Thanks!
